tests_dir/logic/test_trade_logic.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from myapp import models
+from myapp.logic import trade_logic
+
+
+
+class TestTradeLogic(TestCase):
+    def setUp(self):
```

Comment:
this is the same setup as TestRegisterLogic
see if you can reformat for less duplicate code